examples/todo-mvc-svelte/README.md

```diff
@@ -0,0 +1,13 @@
+# Todo MVC example
+
+[Open in CodeSandbox](https://codesandbox.io/embed/github/statelyai/xstate/tree/main/examples/todo-mvc-svelte)
```

Comment:
This link supposed to work hopefully after merge :crossed_fingers:

Till then this link works: https://codesandbox.io/embed/github/annaghi/xstate/tree/examples/todo-mvc-svelte/examples/todo-mvc-svelte